spec/report_spec.rb

```diff
@@ -1045,6 +1045,15 @@ def gloops
     expect(Bugsnag).not_to have_sent_notification
   end
 
+  it 'uses an appropriate message if nil is notified' do
+    Bugsnag.notify(nil)
+    expect(Bugsnag).to have_sent_notification{ |payload, headers|
+      event = payload["events"][0]
+      exception = event["exceptions"][0]
+      expect(exception["errorClass"]).to eq("RuntimeError")
+      expect(exception["message"]).to eq(Bugsnag::Report::NIL_EXCEPTION_DESCRIPTION)
```

Comment:
What does the stacktrace look like in this case? It would be good to verify that it is also sensible in a test.

Reply:
I've updated the test to look at the stacktrace as well now.